app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/controller/SearchController.java

```diff
@@ -92,4 +107,76 @@ public static SearchController getInstance() {
         return resultFuture;
     }
 
+    public Book searchIsbn(String isbn) {
+        AsyncTask<String, Void, String> bookSearch = new SearchGoogleBooks().execute(isbn);
+
+        return isbnBook;
```

Comment:
Why is this not ComparableFuture? It this is called from UI thread there will be issues

Reply:
AsyncTask is Android specific, used for tasks that need to not run on the UI thread. You're right that completeableFuture works here, but AsyncTask will also do the job